sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureCliCredential.java

```diff
@@ -23,16 +24,20 @@ public class AzureCliCredential implements TokenCredential {
     private static final ClientLogger LOGGER = new ClientLogger(AzureCliCredential.class);
 
     private final IdentityClient identityClient;
+    private final IdentitySyncClient identitySyncClient;
```

Comment:
Other SDK clients' naming is like XXXClient and XXXAsyncClient. Should this be considered an inconsistent naming?